Tests/SwiftGRPCTests/ServerThrowingTests.swift

```diff
@@ -53,15 +55,28 @@ class ServerThrowingTests: BasicEchoTestCase {
 
 extension ServerThrowingTests {
   func testServerThrowsUnary() {
+    let caughtError: RPCError
     do {
       let result = try client.get(Echo_EchoRequest(text: "foo")).text
       XCTFail("should have thrown, received \(result) instead")
+      return
     } catch {
-      guard case let .callError(callResult) = error as! RPCError
-        else { XCTFail("unexpected error \(error)"); return }
-      XCTAssertEqual(.permissionDenied, callResult.statusCode)
-      XCTAssertEqual("custom status message", callResult.statusMessage)
+      caughtError = error as! RPCError
```

Comment:
Should we fail the test if this is not an `RPCError`? i.e., `catch let as RPCError { ... }`... `catch { XCTFail(...) }`

Reply:
Good idea! Done.